README.md

```diff
@@ -33,11 +33,19 @@ Options:
   -r, --rate-limit-enabled  Enable rate limiting                       [boolean]
   -c, --cors                CORS whitelist origins                [string]
   -a, --auth                Enable authentication and authorization [boolean]
+  -js, --jwtsecret          JWT Secret                               [string]
```

Comment:
For these two new params @AbegaM - I think you need to clarify that the user is `specifying` the values.